docs/api.md

```diff
@@ -3253,6 +3254,29 @@ As the vast majority of clients only support the JSON OData format, that is the
 + Response 501 (application/json)
     + Attributes (Error 501)
 
+### Data Download Path [GET /dl/projects{projectId}/forms/{xmlFormId}/submissions/{instanceId}/attachments/{filename}]
+
+_(introduced: version 1.2)_
+
+This route is a web browser oriented endpoint intended for user-interactive usage only. It's not part of the Central API, but is documented here as it can be useful.
+
+If you are writing or configuring an OData client and have submission media files to deal with, you can run into authentication problems directly fetching or linking the media file URLs that are provided in the OData feed. This can be due to several reasons: if the user is not logged into the Central administration site (and thus has no valid cookie), if the request comes from a foreign origin (and thus cookies are not sent by the browser), and more.
+
+To help manage this, the frontend provides a `/dl` path that allows file download. Just take a normal attachment download path and replace the `/v1` near the beginning of the path with `/dl`, and the user will be taken to a page managed by the Central administration website that will ensure the user is logged in, and offer the file as a download.
```

Comment:
The path prefix is actually the very beautiful `/#/dl`.

Reply:
😭 I was pleased reading the docs to learn that just `/dl` was supported. Oops!